src/app/form/_utils/fillPdf/ffsIndividual/page9.test.ts

```diff
@@ -7,6 +7,16 @@ describe("Page 9 - SECTION II – PROVIDER IDENTIFICATION", () => {
   const testedPdfKeys = new Set<keyof PdfFfsIndividualPage9>([]);
 
   describe("when NEXT_PUBLIC_FLAG_LEGAL is not set", () => {
+    it("does not fill 1. approved as a provider of services under Medicaid", () => {
+      const yesPdfKey = "fd425approvedprovideryes";
+      const noPdfKey = "fd425approvedproviderno";
+      const explanationPdfKey = "fd425approvedprovideryesexplaination";
```

Comment:
[dust] Pardon me if this came up already: Do these values have to be all lowercase? It makes them hard to read.